srcpkgs/Aegisub/template

```diff
@@ -1,34 +1,43 @@
 # Template file for 'Aegisub'
 pkgname=Aegisub
-version=3.3.2
-revision=11
-build_style=cmake
-build_helper=cmake-wxWidgets-gtk3
-cmake_builddir="BUILD"
-configure_args="-DwxWidgets_CONFIG_EXECUTABLE=$WX_CONFIG
- -DWITH_PORTAUDIO=$(vopt_if portaudio ON OFF)
- -DWITH_OPENAL=$(vopt_if openal ON OFF) -DWITH_FFTW3=$(vopt_if fftw ON OFF)"
-hostmakedepends="gettext-devel intltool pkg-config git LuaJIT"
-makedepends="alsa-lib-devel boost-devel libcurl-devel fontconfig-devel
- freetype-devel hunspell-devel icu-devel libass-devel libffms2-devel
- MesaLib-devel wxWidgets-gtk3-devel LuaJIT-devel $(vopt_if fftw fftw-devel)
- $(vopt_if openal libopenal-devel) $(vopt_if portaudio portaudio-devel)"
-short_desc="Subtitle editor (ssa, ass, srt)"
-maintainer="Orphaned <orphan@voidlinux.org>"
-license="BSD-3-Clause, MIT"
-homepage="https://github.com/wangqr/Aegisub"
-distfiles="https://github.com/wangqr/Aegisub/archive/v${version}.tar.gz"
-checksum="9aaab8e66cef2a72368c6b7f79b2f99700321a7b9a1ca623c5e6be0e65418db5"
-
-build_options="fftw openal portaudio"
-build_options_default="fftw"
-
-desc_option_fftw="Enable support for rendering of audio waveforms/spectrum"
-desc_option_openal="Enable support for openal"
+version=3.4.2
+revision=1
+build_style=meson
+configure_args="-Dsystem_luajit=true"
+hostmakedepends="pkg-config git gettext meson ninja LuaJIT wxWidgets-gtk3-devel"
```

Comment:
You should still strip `wxWidgets-gtk3-devel` from `hostmakedepends`, because the idea is to avoid running the host script.